test/test_kms_with_cli.py

```diff
@@ -132,9 +132,24 @@ def no_bool_convert(pairs):
 
 def test_GenerateQuote_and_VerifyQuote(base_url, headers):
     print('====================test_GenerateQuote_and_VerifyQuote start===========================')
+    mrenclave = '870c42c59bc74c7ad22869411709e4f78ac3c76add6693bb43296b03362e5038';
+    mrsigner = 'c30446b4be9baf0f69728423ea613ef81a63e72acf7439fa0549001fd5482835';   
+
+    policyId = uploadQuotePolicy.uploadQuotePolicy(base_url, mrenclave, mrsigner)
+
+    result_getQuotePlicy = getQuotePolicy.getQuotePolicy(base_url, policyId)
+    if ('mrenclave' in result_getQuotePlicy):
+        print('check getQuotePolicy result with %s: %s' %('mrenclave', mrenclave == result_getQuotePlicy['mrenclave']))
+    else:
+        print('check getQuotePolicy result with %s: %s' %('mrenclave', False))
+    if ('mrsigner' in result_getQuotePlicy):
+        print('check getQuotePolicy result with %s: %s\n' %('mrsigner', mrsigner == result_getQuotePlicy['mrsigner']))
+    else:
+        print('check getQuotePolicy result with %s: %s\n' %('mrsigner', False))
+
     generate_quote.generate_quote_with_file(base_url, "a.txt")
 
-    verify_quote.verify_quote_with_file(base_url, "a.txt")
+    verify_quote.verify_quote_with_file(base_url, "a.txt", policyId)
```

Comment:
which quote did you test? a.txt quote or policyId quote?

Reply:
use an independent code for policyId test. Do not change verify quote with file.

Reply:
The policyId is require parameter.